### DISCUSSION TITLE
ComfyUI管理器Error

### DISCUSSION THREAD

**Discussion by 17634372392** [Comfy-Org/ComfyUI#4134] [Q&A]
使用通过Git URL安装提示This action is not allowed with this security level configuration.

**Comment by clia0** [comment]
https://comfyui-guides.runcomfy.com/ultimate-comfyui-how-tos-a-runcomfy-guide/how-to-fix-this-action-is-not-allowed-with-this-security-level-configuration 按照这里的提示,修改config.ini文件
